### ISSUE TITLE
Feature-Request: Use paging for showing the recorded conversion rather then files

### ISSUE THREAD

**Issue by Haider8** [Haider8/tmessage#36]
Have to be done after #35 is fixed.

**Comment by Haider8** [comment]
This issue has to be fixed as #35 is fixed now.

**Comment by Grommers00** [comment]
Hi,

I would like to attempt this if thats okay?

**Comment by Grommers00** [comment]
Am I thinking incorrectly if that is supposed to say conversation? And you would like it stored in a web version of it?

**Comment by Haider8** [comment]
Yes I meant to say conversation. No we don't want to store it in a web version. Messages are already stored in a sqlite database and we want paging feature now so that users can see their stored messages from the database.

**Comment by Grommers00** [comment]
Ah - so just to confirm in the chat they will say something along the lines of "Get chat - %date" and they will get their conversation showing in the CLI?

**Comment by Haider8** [comment]
For now make it simple, just show the chat which are stored in the database. Order of the messages shown on the cli should be such that recent messages comes first. You can use some library for paging. Please write the code such that it can be utilized in the future in which we have planned to implement personal and group chat feature.

**Comment by Grommers00** [comment]
Hi Haider, I have submitted a rough draft in the pull request. 

It currently grabs the data from the database, and displays it locally. 
I don't think the timestamp order by needs to be added since if you are not online, it will store it in succession.  

There's definitely a lot of potential here, but I think the way messages are stored will need to be saved in conversation form versus the current a message is an entry. 

Also one of the tests failed. I am unsure about that one.

**Comment by Grommers00** [comment]
1596277f40b7a231341db0f751e3301a786aee21

**Comment by Grommers00** [comment]
I think I will pass on this one, as I have zero idea of how actual paging works! Sorry!
